Data_Manipulation/voter_participation_crosswalk.R

```diff
@@ -0,0 +1,229 @@
+# Crosswalk of 2020 Precinct Election Results to ZCTAs 
+# By Caroline Mills 
+# Originated on 12/01/2021 
+# Progress update: 50 states and DC are completed. 
+
+library(sf)
+library(geojsonsf)
+library(tidyverse)
```

Comment:
Please call the libraries you use from the `tidyverse` rather than the whole thing, that could bring in extra dependencies.